docroot/themes/humsci/humsci_basic/src/scss/components/_block-layout.scss

```diff
@@ -38,13 +42,17 @@
 }
 
 .datetime {
-  color: darken($su-color-driftwood, 45%);
+  @include hb-colorful {
```

Comment:
doesnt need to be inside the hb-colorful mixin as that happens in the hb-global-color mixin